queries/phpdoc/highlights.scm

```diff
@@ -0,0 +1,29 @@
+(tag_name) @tag
```

Comment:
I think, `@tag` is not really appropriate here. Even though in phpdoc any `@foo` construct is also called "tag" the `@tag` capture group is reserved for XML tags - which this is not. I'm just not sure what to use instead.

Reply:
Would `@attribute` fit? It's used for C++ attributes (code annotations). The `comment` parsers uses `@text.note`: https://github.com/theHamsta/nvim-treesitter/blob/7d1fb084e2864f8f22e07e61e768561be4dd446f/queries/comment/highlights.scm#L8

Reply:
I've thought about `@attribute`, too. It's at least better than `@tag`. Change pushed.

I'm not sure though if phpdoc can be compared to the comment parser. The latter follows it's own standard for comments whereas phpdoc (like jsdoc) is well defined and has way more features. It would be great if we had something like `@comment.tag`, `@comment.type`, `@comment.variable` etc.